modules/hosted-control-planes-version-support.adoc

```diff
@@ -8,11 +8,32 @@
 [id="hosted-control-planes-version-support_{context}"]
 = Versioning for {hcp}
 
+The {hcp} feature includes the following components, which might require independent versioning an support levels:
+
+* Management cluster
+* HyperShift Operator
+* {hcp-capital} (`hcp`) command-line interface (CLI)
+* `hypershift.openshift.io` API
+* Control Plane Operator
+
+[id="hcp-versioning-mgmt_{context}"]
+== Management cluster
+
+In management clusters for production use, you need {mce}, which is available through OperatorHub. The {mce-short} bundles a supported build of the HyperShift Operator. For your management clusters to remain supported, you must use version of {product-title} that {mce-short} runs on. In general, a new release of {mce-short} runs on the following versions of {product-title}:
+
+* The latest Generally Available version of {product-title}
+* Two versions before the latest Generally Available version of {product-title}
+
+The full list of {product-title} versions that you can install through the HyperShift Operator on a management cluster depends on the version of your HyperShift Operator. However, the list always includes at least the same {product-title} version as the management cluster and two previous minor versions relative to hte management cluster. For example, if the management cluster is running 4.17 and a supported version of {mce-short}, the HyperShift Operator can install 4.17, 4.16, and 4.15 hosted clusters.
```

Comment:
4.17, 4.16, 4.15, and 4.14 hosted clusters

Reply:
> two previous minor versions relative to the management cluster. For example, if the management cluster is running 4.17 and a supported version of multicluster engine Operator, the HyperShift Operator can install 4.17, 4.16, 4.15, and 4.14 hosted clusters.

It may be confusing for the customers, as they are not in an N-2 relationship (4.14 hc for 4.17 HO)?

Reply:
Updating to 4.14-4.17 per conversation with Adel.